lib/crowbar/client/request/base.rb

```diff
@@ -50,7 +50,7 @@ def params
           when :post
             [
               method,
-              content.to_json
+              content
```

Comment:
If it is desired or not depends on the implementation and the request. If not then it has to be changed the same way I did it here. The problem is that we can't change the `base.rb` because this will overwrite all requests which will work in most of the cases but as usual there are exceptions like the upload. 

PUT is always JSON (is only used in the newer models) but POST is still messed up on the server side.